remote/src/lib.rs

```diff
@@ -159,33 +159,40 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
             );
         }
         Operation::Remove { remote } => {
+            let mut remote_found = None;
```

Comment:
This works, but what if we moved this if/else into a function? It would simplify the control flow a lot, ie. we could do `match find_remote(...)? { ... } `